File: expressions/financial_calculator.py
# Kaleb Carrillo, Financial Calculator pythonS

# print statment that welcomes user tells what the program does
print("Welcomes user this is a fanacial calculator for your needs.")
# ask what their income is (varible an input)
income = float(input("What is your income?\n"))
# ask what their rent is (varible an input)
rent = float(input("What is your rent?\n"))
# ask what their utlitites is (varible an input)
utlitites = float(input("What is your utlitites cost?\n"))
# ask what their groceries is (varible an input)
groceries = float(input("What is groceries cost?\n"))
# ask what their transportation is (varible an input)
transportation = float(input("what is your transportation cost?\n"))
#spedning input
#spend = 10
# calculate savings as 10% of income (income*.1) (varible)
savings = (income*.1)
saving = (savings/income)*100
# calculate sav ing as income-savings-rent-utilites-groceries-transportaion (varible)
spending = income-savings-rent-utlitites-groceries-transportation
spendings = (savings/income)*100
# calculate percent income of utilites (utilites/income *100) (varible)
utlitite = (utlitites/income)*100
# calc rent
rents = (rent/income)*100
# calculate percent income of groceries (groceries/income *100) (varible)
grocerie = (groceries/income)*100

transportations = (transportation/income)*100
# calculate percent income of spending (spending/income *100) (varible)
# Your rent i s $XX.XX which is XX% of your income. (Print)
print("your rent is $",rent, "Which is", rents, "% of your income. \n")
# Your utilites is $XX.XX which is XX% of your income. (Print)
print("your utlitites $", utlitites, "Which is", utlitite, "% of your income. \n")
# Your groceries is $XX.XX which is XX% of your income. (Print)
print("Your groceries $", groceries, "Which is", grocerie, "% of your income. \n")
# Your transportation is $XX.XX which is XX% of your income. (Print)
print("Your transportation $", transportation, "Which is", transportations, "% of your income. \n")
# Your spending is $XX.XX which is XX% of your income. (Print)
print("Your spending $", spending, "which is", spendings, "% of your income. \n")
# Your savings is $XX.XX which is XX% of your income. (Print)
print("Your savings $", saving,"which is",savings, "% of your income. \n")

File: expressions/financial_calculator.c
// Kaleb Carrillo, Financial Calculator C

#include <stdio.h>

float income;
float rent;
float utilities;
float groceries;
float transportation;

int main(void){
    printf("Welcomes user this is a fanacial calculator for your needs. \n");
    printf("how much is your in income: \n", income);
    scanf("%f", &income);
    printf("How much is your rent: \n", rent);
    scanf("%f", &rent);
    printf("How much are your utlitiles: \n", utilities);
    scanf("%f", &utilities);
    printf("How much is are your groceries: \n");
    scanf("%f", &groceries);
    printf("How much is transportation: \n", transportation);
    scanf("%f", &transportation);

    float utilitie = (utilities/income)*100;;
    float rents = (rent/income)*100;
    float grocerie = (groceries/income)*100;
    float transportations = (transportation/income)*100;
    float saving = (income*.1);
    float saving_percentage = (saving/income)*100;
    float spending = income-rent-utilities-groceries-transportation-saving;
    float spend = (spending/income)*100;
    
   //calculate sav ing as income-savings-rent-utilites-groceries-transportaion (varible)
    
    printf("your rent is $ %.2f. Which is %.2f %% of your income. \n", rent, rents);
    printf("Your utilities $ %.2f. Which is %.2f %% of your income. \n", utilities, utilitie);
    printf("Your groceries $ %.2f. Which is %.2f %% of your income. \n", groceries, grocerie);
    printf("Your transportation $ %.2f. Which is %.2f %% of your income. \n", transportation, transportations);
    printf("Your have $ %.2f left to spend. Which is %.2f %% of your income. \n", spending, spend);
    printf("Your savings is $ %.2f Which is %.2f percent of your income. \n", saving, saving_percentage);
     return 0;
}